packstream/packstream-specification-1.md

```diff
@@ -517,13 +518,13 @@ Example:
 ## Structure
 
 
-A structure is a composite value, comprised of fields and a unique type code.
+A structure is a composite value, comprised of a unique type code, and a sequence of fields.
```

Comment:
I'm bad with punctuation. Does the comma before "and" belong there?

Reply:
![Screenshot 2021-03-23 at 17 23 13](https://user-images.githubusercontent.com/445792/112181154-74a63080-8bfc-11eb-9d11-8b76ec2b36c1.png)

Reply:
So then a “sequence of fields” is not part of the “composite value”? That seems dubious…

Reply:
oh, wait, those are not independent clauses indeed :)